mixin/build.go

```diff
@@ -104,6 +107,20 @@ func mergeMoMs(mixWS string, mixVer, lastVer int) error {
 	return upstreamMoM.WriteManifestFile("Manifest.MoM")
 }
 
+func incrementMixVerIfNeeded(mixVer int, mixFlagFile string) int {
+	out, err := ioutil.ReadFile(mixFlagFile)
+	if err != nil {
+		return mixVer
+	}
+
+	ver, err := strconv.ParseInt(strings.TrimSpace(string(out)), 10, 32)
```

Comment:
I would tend to use strconv.Atoi(strings.TrimSpace(out)) if I didn't need to be able to specify the base.

If you are sticking with ParseInt then specify the bit size as 0, so the conversion returns an "int" rather than an "int32", which would also remove the need for a cast to int.

Just possibly consider strconv.ParseUInt?

Reply:
Ah, `Atoi` is a much better choice, thank you.

Reply:
done